altos-rust/port/cortex-m0/libs/arm/src/math.rs

```diff
@@ -37,31 +37,72 @@ pub extern "C" fn __aeabi_lmul(a: u64, b: u64) -> u64 {
 
 #[no_mangle]
 pub extern "C" fn __aeabi_uidiv(num: u32, den: u32) -> u32 {
-  __aeabi_uidivbase(num, den, false)
+    __aeabi_uidivbase(num, den, false)
 }
 
 #[no_mangle]
-pub extern "C" fn __aeabi_uidivmod(num: u32, den: u32) -> u32 {
-  __aeabi_uidivbase(num, den, true)
+pub extern "C" fn __udivmodsi4(mut num: u32, mut den: u32, rem_p: Option<&mut u32>) -> u32 {
+	let mut quot = 0;
```

Comment:
This file's indentation level does not appear to be offset at 4 spaces at each level.